tests/transition_to_if.py

```diff
@@ -16,11 +16,10 @@
 errorExit=Utils.errorExit
 
 appArgs = AppArgs()
-args=TestHelper.parse_args({"-p","-d","-s","--keep-logs","--dump-error-details","-v","--leave-running","--unshared"},
+args=TestHelper.parse_args({"-p","-d","--keep-logs","--dump-error-details","-v","--leave-running","--unshared"},
                             applicationSpecificArgs=appArgs)
 pnodes=args.p if args.p > 4 else 4
 delay=args.d
-topo=args.s
```

Comment:
The test is hard-coded for the expected topo. A user provided one would not work.